tests/helpers/test_template.py

```diff
@@ -229,41 +230,41 @@ def test_rounding_value(hass):
         template.Template(
             "{{ states.sensor.temperature.state | round(1) }}", hass
         ).async_render()
-        == "12.8"
+        == 12.8
     )
 
     assert (
         template.Template(
             "{{ states.sensor.temperature.state | multiply(10) | round }}", hass
         ).async_render()
-        == "128"
+        == 128
     )
 
     assert (
         template.Template(
             '{{ states.sensor.temperature.state | round(1, "floor") }}', hass
         ).async_render()
-        == "12.7"
+        == 12.7
     )
 
     assert (
         template.Template(
             '{{ states.sensor.temperature.state | round(1, "ceil") }}', hass
         ).async_render()
-        == "12.8"
+        == 12.8
     )
 
     assert (
         template.Template(
             '{{ states.sensor.temperature.state | round(1, "half") }}', hass
         ).async_render()
-        == "13.0"
+        == 13.0
     )
 
 
 def test_rounding_value_get_original_value_on_error(hass):
     """Test rounding value get original value on error."""
-    assert template.Template("{{ None | round }}", hass).async_render() == "None"
+    assert template.Template("{{ None | round }}", hass).async_render() is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)